### ISSUE TITLE
iron-component-page requries hydrolysis

### ISSUE THREAD

**Issue by e111077** [Polymer/polymer-analyzer#342]
Heya, I was upgrading `iron-component-page` to the 2.0 variant and found that it relies on hydrolysis analyzer as a polymer element. Do you guys have any insight on how I should handle this migration with the new polymer-analyzer?

**Comment by rictic** [comment]
Once https://github.com/Polymer/polymer-cli/pull/426 lands you'll be able to run `polymer analyze` on a file to get an `elements.json` file, which can be consumed by `iron-component-page` v2. I think we'd then want to add automagic elements.json generation and serving to `polymer serve` for the local dev case, and otherwise you'd serve out a pregenerated `elements.json` file.

We aren't writing off the option of running the analyzer in a browser, but it's not a priority in the short term.

**Comment by e111077** [comment]
Thanks looking forward to that PR landing 👍

**Comment by rictic** [comment]
Also filed https://github.com/PolymerLabs/polyserve/issues/106 to track automagic generation in `polymer serve`
